packages/app/src/app/hooks/useWorkspaceLimits.ts

```diff
@@ -49,7 +49,7 @@ export const useWorkspaceLimits = (): WorkspaceLimitsReturn => {
   const isCloseToOutOfCredits =
     isFree === true && !frozen && usage.credits > 350; // Random pick for now
   const isAtSpendingLimit = isPro === true && frozen; // TODO
```

Comment:
Are we okay to merge without implementing these two conditionals above?

Reply:
the spending limit is actually ok like this, I will remove the TODO. the other is set to false until we have the exact credit usage from the BE and we can show the right banner